inlong-sort/pom.xml

```diff
@@ -35,18 +35,22 @@
         <module>sort-api</module>
         <module>sort-common</module>
         <module>sort-formats</module>
-        <module>sort-connectors</module>
         <module>sort-core</module>
+        <module>sort-flink</module>
         <module>sort-dist</module>
         <module>sort-end-to-end-tests</module>
     </modules>
     <properties>
         <debezium.version>1.5.4.Final</debezium.version>
         <kafka.clients.version>2.7.0</kafka.clients.version>
-        <rat.basedir>${basedir}</rat.basedir>
         <hbase.version>2.2.3</hbase.version>
-        <iceberg.hive.version>2.3.7</iceberg.hive.version>
         <hudi.hive.version>2.3.7</hudi.hive.version>
+        <iceberg.hive.version>2.3.7</iceberg.hive.version>
+        <flink.jackson.version>2.12.1-13.0</flink.jackson.version>
+        <flink.cdc.base.version>2.3.0</flink.cdc.base.version>
+        <log4j.configurationfile>log4j2-test.properties</log4j.configurationfile>
+        <oracle.jdbc.version>19.3.0.0</oracle.jdbc.version>
+        <rat.basedir>${basedir}</rat.basedir>
```

Comment:
ditto

Reply:
Sort is mainly about the flink connector related management in sort, and other shared configurations have been moved to the root pom.xml.